test/test_helper.rb

```diff
@@ -4,15 +4,15 @@
 
 class ActiveSupport::TestCase
   # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
-  fixtures :all     
+  fixtures :all
 
   Time.zone = 'Alaska'
   Time.now
   Date.today
-begin
-  
-rescue Exception
-  
-end
+  begin
+
+  rescue Exception
```

Comment:
__[staging][rails_best_practices]__
* not rescue Exception [:ref](http://rails-bestpractices.com/posts/2012/11/01/don-t-rescue-exception-rescue-standarderror/)